SuggestSearchIT.testEmpty failure in CI

Failed in master for the first time today, does not reproduce

```
java.lang.AssertionError: [shard [[4C1bk--TT7KuHqh-NWQx9g][test][1]], reason [RemoteTransportException[[node_s0][127.0.0.1:36557][indices:data/read/search[phase/query]]]; nested: IllegalArgumentException[no mapping found for field [text]]; ], cause [java.lang.IllegalArgumentException: no mapping found for field [text]
	at org.elasticsearch.search.suggest.SuggestionBuilder.populateCommonFields(SuggestionBuilder.java:307)
	at org.elasticsearch.search.suggest.term.TermSuggestionBuilder.build(TermSuggestionBuilder.java:448)
	at org.elasticsearch.search.suggest.SuggestBuilder.build(SuggestBuilder.java:175)
	at org.elasticsearch.search.SearchService.parseSource(SearchService.java:808)
	at org.elasticsearch.search.SearchService.createContext(SearchService.java:607)
	at org.elasticsearch.search.SearchService.createAndPutContext(SearchService.java:582)
	at org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:385)
```

Reproduce line:
```
REPRODUCE WITH: ./gradlew :server:integTest \
  -Dtests.seed=7E73F9B87A524CD5 \
  -Dtests.class=org.elasticsearch.search.suggest.SuggestSearchIT \
  -Dtests.method="testEmpty" \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SA \
  -Dtests.timezone=America/Louisville \
  -Dcompiler.java=11 \
  -Druntime.java=8
```

Log:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+internalClusterTest/1711/console

--- a/server/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java
+++ b/server/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java
@@ -83,7 +83,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
 
     // see #3196
     public void testSuggestAcrossMultipleIndices() throws IOException {
-        prepareCreate("test").addMapping("type1", "text", "type=text").get();
+        assertAcked(prepareCreate("test").addMapping("type1", "text", "type=text"));
         ensureGreen();
 
         index("test", "type1", "1", "text", "abcd");
@@ -97,7 +97,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
                 .text("abcd");
         logger.info("--> run suggestions with one index");
         searchSuggest("test", termSuggest);
-        prepareCreate("test_1").addMapping("type1", "text", "type=text").get();
+        assertAcked(prepareCreate("test_1").addMapping("type1", "text", "type=text"));
         ensureGreen();
 
         index("test_1", "type1", "1", "text", "ab cd");
@@ -302,7 +302,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
     }
 
     public void testSimple() throws Exception {
-        prepareCreate("test").addMapping("type1", "text", "type=text").get();
+        assertAcked(prepareCreate("test").addMapping("type1", "text", "type=text"));
         ensureGreen();
 
         index("test", "type1", "1", "text", "abcd");
@@ -327,7 +327,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
     }
 
     public void testEmpty() throws Exception {
-        prepareCreate("test").addMapping("type1", "text", "type=text").get();
+        assertAcked(prepareCreate("test").addMapping("type1", "text", "type=text"));
         ensureGreen();
 
         index("test", "type1", "1", "text", "bar");
@@ -346,7 +346,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
     }
 
     public void testWithMultipleCommands() throws Exception {
-        prepareCreate("test").addMapping("typ1", "field1", "type=text", "field2", "type=text").get();
+        assertAcked(prepareCreate("test").addMapping("typ1", "field1", "type=text", "field2", "type=text"));
         ensureGreen();
 
         index("test", "typ1", "1", "field1", "prefix_abcd", "field2", "prefix_efgh");

--- a/server/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java
+++ b/server/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java
@@ -83,7 +83,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
 
     // see #3196
     public void testSuggestAcrossMultipleIndices() throws IOException {
-        createIndex("test");
+        prepareCreate("test").addMapping("type1", "text", "type=text").get();
         ensureGreen();
 
         index("test", "type1", "1", "text", "abcd");
@@ -97,7 +97,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
                 .text("abcd");
         logger.info("--> run suggestions with one index");
         searchSuggest("test", termSuggest);
-        createIndex("test_1");
+        prepareCreate("test_1").addMapping("type1", "text", "type=text").get();
         ensureGreen();
 
         index("test_1", "type1", "1", "text", "ab cd");
@@ -302,7 +302,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
     }
 
     public void testSimple() throws Exception {
-        createIndex("test");
+        prepareCreate("test").addMapping("type1", "text", "type=text").get();
         ensureGreen();
 
         index("test", "type1", "1", "text", "abcd");
@@ -327,7 +327,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
     }
 
     public void testEmpty() throws Exception {
-        createIndex("test");
+        prepareCreate("test").addMapping("type1", "text", "type=text").get();
         ensureGreen();
 
         index("test", "type1", "1", "text", "bar");
@@ -346,7 +346,7 @@ public class SuggestSearchIT extends ESIntegTestCase {
     }
 
     public void testWithMultipleCommands() throws Exception {
-        createIndex("test");
+        prepareCreate("test").addMapping("typ1", "field1", "type=text", "field2", "type=text").get();
         ensureGreen();
 
         index("test", "typ1", "1", "field1", "prefix_abcd", "field2", "prefix_efgh");

